.github/ISSUE_TEMPLATE/harness.yml

```diff
@@ -0,0 +1,57 @@
+name: Test Harness
```

Comment:
question/discussion:

Should we have this as "Tests" instead?

Reply:
I think "Tests" would get confusing for the automated tests of the LSP plugin (such as unit and e2e tests) I feel that "Test Harness" is more explicit. It might not initially be obvious what it is to people but 
1. I explain what it is fairly thoroughly in the description
2. If someone doesn't know what it is then it is very likely they shouldn't be using this issue template anyway

Going to leave this as it is